src/mscorlib/src/System/Lazy.cs

```diff
@@ -50,63 +42,22 @@ internal static class LazyHelpers
     [ComVisible(false)]
     [DebuggerTypeProxy(typeof(System_LazyDebugView<>))]
     [DebuggerDisplay("ThreadSafetyMode={Mode}, IsValueCreated={IsValueCreated}, IsValueFaulted={IsValueFaulted}, Value={ValueForDebugDisplay}")]
-    public class Lazy<T>
+    public class Lazy<T> :
```

Comment:
Hi @manofstick, I have checked the code and added a few observations/questions. Overall I see no obvious weakness. You said there was no test-coverage for `Lazy<'T>` today. Can that be added (damn difficult for these kinds of classes but at least to cover the functionality)?